Submitted By: Mario Fetka (mario-fetka at gmx dot at)
Date: 2008-07-30
Initial Package Version: 1.14.17.1
Origin: me
Upstream Status: unknown
Description: make sample compile with gcc 4.3
 
diff -Naur work.orig/usr/share/doc/lightscribe-sdk/sample/lsprint/lsprint.cpp work/usr/share/doc/lightscribe-sdk/sample/lsprint/lsprint.cpp
--- work.orig/usr/share/doc/lightscribe-sdk/sample/lsprint/lsprint.cpp	2008-06-09 19:16:18.000000000 +0000
+++ work/usr/share/doc/lightscribe-sdk/sample/lsprint/lsprint.cpp	2008-07-30 18:17:33.000000000 +0000
@@ -23,6 +23,8 @@
 // Copyright (C) 2005-2006 Hewlett-Packard Development Company, LP
 // 
 
+#include <cstdlib>
+#include <cstring>
 
 // import LSAPI
 #include "lightscribe.h"
